addon/components/search-result/template.hbs

```diff
@@ -2,6 +2,13 @@
 <div class="search-result p-sm">
     <div class="row">
         <div class="col-xs-12">
+            {{!ShareResult}}
+            {{sharing-icons-popover
```

Comment:
since the result is passed in as argument to share-result component, I think it's safer to have a check for result, that way if ever result is `undefined`, we don't get TypeErrors.